buildifier/runner.bash.template

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+BUILDIFIER_SHORT_PATH=@@BUILDIFIER_SHORT_PATH@@
+ARGS=@@ARGS@@
+
+buildifier_short_path=$(readlink "$BUILDIFIER_SHORT_PATH")
+cd "$BUILD_WORKSPACE_DIRECTORY"
```

Comment:
This is using new Bazel 0.12 env variable. See https://blog.bazel.build/2018/04/11/bazel-0.12.html